src/Dockerfile

```diff
@@ -11,7 +11,7 @@ COPY . $WORKDIR
 # installing dependencies to build package
 RUN pip install . -t python
 
-# change 31812
+# change 31815
```

Comment:
I forget. Why is this necessary? I seem to remember it having to do with Terraform. 🤔

Reply:
The reasons are slightly unclear why (to me, who hasn't tried to figure it out in a long while), but basically if there isn't a change in the Dockerfile (or its hash, most likely) then Terraform doesn't think it needs to replace a particular resource (which one, I forget, but it's important) in AWS. It's annoying, but we are lazy. :)